### ISSUE TITLE
Mac VoiceOver issue with focus moving when activating an HTML Summary/Details element

### ISSUE THREAD

**Issue by clapierre** [edrlab/thorium-reader#1393]
This is with Thorium 1.6.0 but the same behavior was in older versions as well.

Here is a [video showing this problem](https://benetech.zoom.us/rec/share/247iwLcQ96fMAoE1nt8SY6WM3hNVFnZ6XUKuCU1ZMT6eMCFarc9XgVsLQMPFTt8B.GRCUCxp_-iDO1h7g ) where we have the [epubtest.org EPUB book for Extended Descriptions v1.1.1](http://epubtest.org/books/Accessibility-Tests-Extended-Descriptions-v1.1.1.epub) 

When VoiceOver on a Mac is turned on and you activate a Summary/Details element expanding the details the focus jumps up the page which seems to be screen size dependent on how far the focus jumps based on the size of the content in the details.

The focus should stay on the carrot summary and ideally that element shouldn't move on the page either only content lower should move further down the scrollview. 

So even without VoiceOver turned on and you activate this summary to expand the details the position of that summary text moves up the screen.  So if a low vision users is zoomed in, they may loose their position and orientation as the text has move up the screen.

Ideally the summary would stay anchored wherever the user activates it and the content in the details would appear below that anchor point increasing the scrollable area below that anchor point until the user manually moves their focus into that details content.

**Comment by danielweck** [comment]
I have just tested this with Voice Over in Thorium 2.1.0, I am unable to reproduce the problem. Closing as solved.
